src/main/index.js

```diff
@@ -1015,6 +1015,45 @@ function runApp() {
     }
   })
 
+  // *********** //
+  // Highlighted Comments
+  ipcMain.handle(IpcChannels.DB_HIGHLIGHTED_COMMENTS, async (event, { action, data }) => {
+    console.error('mario index.js cases')
```

Comment:
Don't leave test messages lying around. Also if you do add test messages, use `console.log`, that way the linter will complain about it before letting you commit, so you'll remember to remove it.